Bounce/dist/presence.js

```diff
@@ -117,8 +117,30 @@ presence.on("UpdateData", () => {
     }
      else if(document.location.pathname.startsWith("/portal")) {
       presenceData.state = "Browsing Staff Panel";
-      presenceData.details = "Logging in"
-  }
+      presenceData.details = "Logging in" 
+     
+         break;
+         case "user.home":
+            presenceData.details = "Viewing Homepage"
+         break;
+         case "user.rules":
+            presenceData.details = "Viewing Rules"
+         break;
+         case "user.bookaway":
+            presenceData.details = "Booking Away"
+         break;
+         case "news.create":
+            presenceData.details = "Creating Article"
+         break;
+         case "editor.allstories":
+            presenceData.details = "Viewing Articles"
+         break;
+         case "manager.modifyaccounts":
+            presenceData.details = "Updating Account"
+         break;
+         case "manager.createaccount":
+            presenceData.details = "Creating Account"                  
```

Comment:
Where is the switch? xD

Reply:
Tbh, Our Devs away so it'll take me a while to figure out what ya mean by that lool

Reply:
I'm assuming it's something like? ' switch (document.location.hash.replace("/portal/", "").replace("#", "")) { '